doc/sr_python_walkthrough.rst

```diff
@@ -67,27 +65,25 @@ Scripts
 =======
 
 Scripts are a way to store python-executable code in the database. The Python
-client can send scripts to the dataset from a file, or directly from memory.
+client can send scripts to the dataset from a file or directly from memory.
 
-As an example, the code below illustrates how a function can be defined and sent
-to the database on the fly, without storing it in an intermediate file.
+The code below illustrates how to avoid storing a function in an intermediate file. 
+With this technique, we can define and send a function to the database on the fly.
 
 .. literalinclude:: ../smartredis/examples/serial/python/example_script.py
   :language: python
   :linenos:
   :lines: 26-66
 
-The code below shows how to set a script from a file.  Running the
-script set from file uses the same API calls as the example shown
-above.
+The code below shows how to set a script from a file. Running the script set from 
+the file uses the same API calls like the example shown above.
 
 .. literalinclude:: ../smartredis/examples/serial/python/example_script_file.py
   :language: python
   :linenos:
   :lines: 26-41
 
-The content of the script file has to be written
-in Python. For the example above, the file ``data_processing_script.txt``
+The content of the script file requires Python. For the example above, the file ``data_processing_script.txt``
```

Comment:
This was originally awkwardly worded, but the rewrite also sounds a little odd in that it decouples Python from the inherent essence of what the file is. Perhaps change to: `This file must be a valid Python script`

Reply:
Oop it does sound odd, I will investigate